src/devhub/components/molecule/Input.jsx

```diff
@@ -121,6 +128,17 @@ const TextInput = ({
 
   const onKeyDown = props.onKeyDown ?? (() => {});
 
+  const getFormattedData = useCallback(() => {
+    if (inputProps.inputmode === "numeric") {
+      const number = parseFloat(convertToString(state.data).replace(/,/g, ""));
+      if (!isNaN(number)) {
+        // Format the number for display
+        return number.toLocaleString("en-US");
```

Comment:
Do we want to use the browsers locale here instead? In most of Europe I believe we use spaces as thousand separator, and comma as decimal separator.

Reply:
yeah I tried using that
it's using `navigator.language` in react webapp, but in VM `navigator` returns undefined, do you know any other way?

Reply:
how about this?

<img width="906" alt="image" src="https://github.com/NEAR-DevHub/neardevhub-bos/assets/9760441/268de514-cdb4-4d51-bb13-ef22867058e9">

Reply:
I think we should "US" locale itself, since in other places for amount, dates Ori asked to shift to US format for consistency in legal process.
If we incorporate all locales, it will lead to more possibility of errors, what do you think?

Reply:
That's a good point. Let's lock to US then.